drafthorse/xmp_schema.py

```diff
@@ -0,0 +1,86 @@
+"""
+FACTUR-X XMP with the required PDF/A extension schema description
```

Comment:
Now that we have this, we should be able to delete `schemas/ZUGFeRD2p2_extension_schema.xmp`